app.config

```diff
@@ -1,6 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <appSettings>
-    <add key="FetBinaryLocation" value="lib/fet/fet-cl" />
-  </appSettings>
+  <configSections>
+    <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
+  </configSections>
+  <connectionStrings>          
+    <add name="DataModel" connectionString="data source=192.168.0.1;initial catalog=SMSV2;persist security info=True;user id=SMSV2;MultipleActiveResultSets=True;App=EntityFramework;User ID=SMSV2;Password=SMSV2" providerName="System.Data.SqlClient" /></connectionStrings>
```

Comment:
Remove password from config file.

Reply:
I don't know how to fix this right, do you?

Reply:
Yes, fixed it.